tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+        let response = putEU('dialogs/' + dialogId + '/systemlabels', body);
+        expectStatusFor(response).to.equal(204);
+        response = getEU('dialogs/' + dialogId + '/labellog');
+        expectStatusFor(response).to.equal(200);
+        expect(response, 'response').to.have.validJsonBody();
+        console.log(response.json());
+        expect(response.json(), 'response body').to.have.lengthOf(1);
+    });
+
+    describe('Changing labels trigger 2 logs', () => {
+
+        let body = {
```

Biome

[error] This let declares a variable that is only assigned once.

'body' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)